software/grimoire.yml

```
@@ -0,0 +1,11 @@
+name: "Grimoire"
+website_url: "https://grimoire.pro"
+source_code_url: "https://github.com/goniszewski/grimoire"
+description: "Bookmark manager with a modern UI, automatic content & metadata extraction, categorization, filtering, and more. It has fully documented REST API, and Docker image for easy deployment."
+licenses:
+  - MIT
+platforms:
+  - Nodejs
+  - Docker
```

Looking through https://grimoire.pro/docs/getting-started/quick-start and also the read-me, there only seems to be install instructions for Docker so lets remove Nodejs.

```suggestion
platforms:
  - Docker
```

---

The app can also be run using the `./run-dev.sh` script (as stated in the [Development document](https://grimoire.pro/docs/getting-started/development/)). But if it's an issue, I can remove `Nodejs` as a platform.

---

Platforms are usually the end user running platforms rather than development. I.e when scrolling on the site, a person may use the platforms as a way of filtering certain projects.

We can probably make this clearer somewhere.

---

@Ki-er I was just in the middle of the `README.md` update, so I've added two more installation instructions that depend on Node (the website documentation will catch up shortly).

Please let me know if there's something I can improve here!

---

Awesome changes, @nodiscc would Pocketbase count as a third party or not? If it does, we would need to readd the depends_3rd party.

---

As far as I know, pocketbase is fully FOS and self-hostable (https://github.com/pocketbase/pocketbase) so I don't think `depends_3rdparty` is required